ac_wrapper.sh

```diff
@@ -27,6 +27,9 @@ fi
 if [ -x "`which dos2unix 2> /dev/null`" ]; then
     dos2unix $ASSET_LIST_FILE
 fi
+CAPTURE_RESOLUTION=${CAPTURE_RESOLUTION:-1280x720}
+export CAPTURE_WIDTH=$(cut -d x -f 1 <<< $CAPTURE_RESOLUTION)
```

Comment:
May as well just set `CAPTURE_RESOLUTION_WIDTH` and `CAPTURE_RESOLUTION_HEIGHT` in ac.rc, instead of `CAPTURE_RESOLUTION`. Would save the need to split with `cut`.